.github/workflows/check-full.yaml

```diff
@@ -12,31 +12,44 @@ jobs:
   R-CMD-check:
     runs-on: ${{ matrix.config.os }}
 
-    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }}) ${{ matrix.config.locale }}
 
     strategy:
       fail-fast: false
       matrix:
         config:
-          - {os: macos-latest,   r: 'release'}
+          - { os: macos-latest, r: "release" }
 
-          - {os: windows-latest, r: 'devel'}
-          - {os: windows-latest, r: 'release'}
+          - { os: windows-latest, r: "devel" }
+          - { os: windows-latest, r: "release" }
           # use 4.1 to check with rtools40's older compiler
-          - {os: windows-latest, r: '4.1'}
+          - { os: windows-latest, r: "4.1" }
 
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
-          - {os: ubuntu-latest,   r: 'release'}
-          - {os: ubuntu-latest,   r: 'oldrel-1'}
-          - {os: ubuntu-latest,   r: 'oldrel-2'}
-          - {os: ubuntu-latest,   r: 'oldrel-3'}
-          - {os: ubuntu-latest,   r: 'oldrel-4'}
+          - { os: ubuntu-latest, r: "devel", http-user-agent: "release" }
+          - { os: ubuntu-latest, r: "devel", locale: "en_US" }
+          #- { os: ubuntu-latest, r: "release", locale: "zh_CN" }
```

Comment:
Some tests fail in this locale, but we can deal with it in a different PR.

Reply:
Do we need `http-user-agent: "release"` here?

Reply:
This is not something I added; it is present in the example actions itself: https://github.com/r-lib/actions/blob/bc8320839a8d0af1e88cadfa533048e49eeee845/examples/check-full.yaml#L33